fw/cfg.c

```diff
@@ -1771,6 +1809,39 @@ tfw_cfg_set_str(TfwCfgSpec *cs, TfwCfgEntry *e)
 	return 0;
 }
 
+int
+tfw_cfg_set_mem(TfwCfgSpec *cs, TfwCfgEntry *e)
+{
+	long *dest_long;
+	unsigned long val;
+	TfwCfgSpecMem *cse;
+
+	BUG_ON(!cs->dest);
+
+	if (tfw_cfg_check_single_val(e))
+		return -EINVAL;
+
+	/* Check value restrictions if we have any in the spec extension. */
+	cse = cs->spec_ext;
+	if (cse) {
+		int r;
+
+		r = tfw_cfg_mem_check_multiple_of(e->vals[0],
```

Comment:
as for me better
if ((r =  tfw_cfg_mem_check_multiple_of(..))
    || (r =  tfw_cfg_mem_check_range()))
     return r;

Reply:
And also may be better to call val = memparse(e->vals[0], NULL); before and then pass val to tfw_cfg_mem_check_multiple_of? Because you call memparse in checking functions two extra times

Reply:
> And also may be better to call val = memparse(e->vals[0], NULL); before and then pass val to tfw_cfg_mem_check_multiple_of? Because you call memparse in checking functions two extra times

No there is no sense, because in `tfw_cfg_mem_check_multiple_of()` we'll have to do "reverse" `memparse` from 
`val` to find correct prefix for error message.

> as for me better
> if ((r = tfw_cfg_mem_check_multiple_of(..))
> || (r = tfw_cfg_mem_check_range()))
> return r;

If you wish, I'll change it, for me the same.